packages/config/src/layer2s/arbitrum.ts

```diff
@@ -515,12 +515,12 @@ export const arbitrum: Layer2 = {
   },
   knowledgeNuggets: [
     {
-      title: 'Arbitrum update increasing decentralization',
+      title: 'Arbitrum update boosts decentralization',
       url: 'https://twitter.com/bkiepuszewski/status/1594754717330309120',
       thumbnailUrl: NUGGETS.BARTEK_TWITTER_THUMBNAIL,
     },
     {
-      title: 'Arbitrum is down - but has it really stopped working?',
+      title: 'Arbitrum is down - but does it work?',
```

Comment:
I think some meaning got lost in your translation... how about: `Arbitrum is down... or is it?` ? 

Also chatGPT for the rescue: 
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/1814312/223419940-38cb794d-072e-454d-aaae-125f566c9f8c.png">

Reply:
Incredible!